<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileEditor7</class>
 <widget class="QDialog" name="ProfileEditor7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 7 Profile Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelProfileName">
     <property name="text">
      <string>Profile Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditProfileName"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelProfileTID">
     <property name="text">
      <string>TID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="TextBox" name="textBoxProfileTID">
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelProfileVersion">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxProfileVersion">
     <item>
      <property name="text">
       <string>Sun</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Moon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ultra Sun</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ultra Moon</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelProfileSID">
     <property name="text">
      <string>SID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="TextBox" name="textBoxProfileSID">
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelProfileTick">
     <property name="text">
      <string>Tick</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="TextBox" name="textBoxProfileTick">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QCheckBox" name="checkBoxShinyCharm">
     <property name="text">
      <string>Shiny Charm</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelProfileOffset">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="TextBox" name="textBoxProfileOffset">
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditProfileName</tabstop>
  <tabstop>comboBoxProfileVersion</tabstop>
  <tabstop>textBoxProfileTick</tabstop>
  <tabstop>textBoxProfileOffset</tabstop>
  <tabstop>textBoxProfileTID</tabstop>
  <tabstop>textBoxProfileSID</tabstop>
  <tabstop>checkBoxShinyCharm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
